app/uk/gov/hmrc/agentclientrelationships/controllers/Relationships.scala

```diff
@@ -19,42 +19,42 @@ package uk.gov.hmrc.agentclientrelationships.controllers
 import play.api.libs.concurrent.Execution.Implicits.defaultContext
 import play.api.libs.json.Json
 import play.api.mvc.Action
-import uk.gov.hmrc.agentclientrelationships.model.Relationship
+import uk.gov.hmrc.agentclientrelationships.model.{Arn, Relationship}
 import uk.gov.hmrc.agentclientrelationships.repositories.RelationshipRepository
-import uk.gov.hmrc.domain.{AgentCode, SaUtr}
 import uk.gov.hmrc.play.microservice.controller.BaseController
 
 import scala.concurrent.Future
 
 class Relationships(relationshipRepository: RelationshipRepository) extends BaseController {
+    val regime = "mtd-sa"
 
-    def create(saUtr: SaUtr, agentCode: AgentCode) = Action.async {
-        findNonRemoved(saUtr, agentCode).flatMap {
-            case Nil => relationshipRepository.create(saUtr.toString(), "sa", agentCode).map(Json.toJson(_)).map(Ok(_))
+    def create(clientRegimeId: String, arn: Arn) = Action.async {
```

Comment:
Maybe the `clientRegimeId` parameters in this class should be renamed to `mtdSaUtr`, because if you pass any other kind of client regime ID this class will do the wrong thing (since regime = "mtd-sa" is hard-coded)?

Alternatively we could get the regime from the URL (I see in app.routes that the URLs have a static `/mtd-sa` component) and pass it in, thus supporting any regime. Probably not worth it at the moment if we don't need support for other regimes.